### ISSUE TITLE
`docker ps --size` reports incorrect value on overlay

### ISSUE THREAD

**Issue by cpuguy83** [docker/docker#13301]
Problem:
Using overlay, `docker ps --size` is reporting a size that is _way_ off.

Reproduction:
```
$ docker run -it -rm alpine
/ # apk update && apk add git # installs git
```
In another terminal run `docker ps --size`.
The reported size here is **229.3MB**

Running `du -h` from `/` in the container shows the total size as **23.9MB**.
Running `docker commit` on the container, and the reported size of the image is **22.81MB**
Note this is _~1/10th_ the reported size from `docker ps --size`

Expected Results:
`docker ps --size` should be reporting what would be the new layer size if the container were committed.

Actual Results:
The reported size is MUCH larger than the actual result.

```
$ docker info
Containers: 1
Images: 573
Storage Driver: overlay
 Backing Filesystem: extfs
Execution Driver: native-0.2
Kernel Version: 4.0.3-boot2docker
Operating System: Boot2Docker 1.6.2 (TCL 5.4); master : 4534e65 - Wed May 13 21:24:28 UTC 2015
CPUs: 8
Total Memory: 3.86 GiB
Name: dev
ID: PSAQ:SH2I:QEUC:DDFR:NEEI:SY64:U3D4:LETV:WQ4L:UX2U:R7UF:3SOM
Debug mode (server): true
Debug mode (client): false
Fds: 12
Goroutines: 17
System Time: Mon May 18 13:08:51 UTC 2015
EventsListeners: 0
Init SHA1: 7f9c6798b022e64f04d2aff8c75cbf38a2779493
Init Path: /usr/local/bin/docker
Docker Root Dir: /mnt/sda1/var/lib/docker
Username: cpuguy83
Registry: [https://index.docker.io/v1/]
Labels:
 provider=virtualbox
```

```
$ docker version
Client version: 1.6.2
Client API version: 1.18
Go version (client): go1.4.2
Git commit (client): 7c8fca2
OS/Arch (client): darwin/amd64
Server version: 1.6.2
Server API version: 1.18
Go version (server): go1.4.2
Git commit (server): 7c8fca2
OS/Arch (server): linux/amd64
```

**Comment by cpuguy83** [comment]
#ESORRYDIDNTINCLUDEUNAMEGORDON

Linux dev 4.0.3-boot2docker #1 SMP Wed May 13 20:54:49 UTC 2015 x86_64 GNU/Linux

**Comment by UserTaken** [comment]
Can confirm 

```
$ docker run --name test alpine sh -c "apk update ; apk add git" ; docker ps -as | grep test

...

OK: 23 MiB in 26 packages
e9acd577dc62        alpine:latest       "sh -c 'apk update ;   11 seconds ago      Exited (0) Less than a second ago       test         229.9 MB
```

but if I run 

```
docker run --name test alpine sh -c "apk update ; apk add htop curl nginx php nmap qemu" ; docker ps -as | grep test

...

OK: 54 MiB in 52 packages
55219f9ff0c9        alpine:latest       "sh -c 'apk update ;   31 seconds ago      Exited (0) Less than a second ago       test         49.17 MB
```

the size looks right.

```
Client version: 1.6.2
Client API version: 1.18
Go version (client): go1.4.2
Git commit (client): 7c8fca2
OS/Arch (client): linux/amd64
Server version: 1.6.2
Server API version: 1.18
Go version (server): go1.4.2
Git commit (server): 7c8fca2
OS/Arch (server): linux/amd64

Containers: 3
Images: 112
Storage Driver: overlay
 Backing Filesystem: extfs
Execution Driver: native-0.2
Kernel Version: 4.0.4-1-ARCH
Operating System: Arch Linux
CPUs: 4
Total Memory: 1.959 GiB
Name: localhost
ID: V7CQ:ZUWZ:UMDJ:N2QE:V65V:WDOJ:ERVZ:I3VT:KDPU:NTBY:4AAQ:WXUJ
```

**Comment by cpuguy83** [comment]
Fixed by #13310
